stdlib/builtins.pyi

```diff
@@ -909,7 +909,7 @@ class list(MutableSequence[_T], Generic[_T]):
 class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     # __init__ should be kept roughly in line with `collections.UserDict.__init__`, which has similar semantics
```

Comment:
Should we make a similar change to `collections.UserDict.__init__`

Reply:
Good point, made similar changes to constructors in that file.